hedera-node/test-clients/build.gradle.kts

```diff
@@ -26,7 +24,7 @@ tasks.register<JavaExec>("runTestClient") {
     group = "build"
     description = "Run a test client via -PtestClient=<Class>"
 
-    classpath = sourceSets.main.get().runtimeClasspath + files(tasks.jar)
+    classpath = configurations.runtimeClasspath.get().plus(files(tasks.jar)) + files(tasks.jar)
```

Comment:
Why is `files(tasks.jar))` added twice?

Reply:
..because copy&paste and I am blind. Good catch! Fixed.